Add-ons/Umbraco-Deploy/Upgrades/version-specific.md

```diff
@@ -0,0 +1,65 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Version specific upgrades"
+meta.Description: "Version specific documentation for upgrading to new major versions of Umbraco Deploy."
+state: complete
+---
+
+# Version Specific Upgrade Details
+This page covers specific upgrade documentation for when migrating to a new major of Umbraco Deploy.
+
+## Version 10
+Version 10 of Umbraco Deploy has a minimum dependency on Umbraco CMS core of `10.0.0`. It runs on .NET 6.
+
+The forms deployment component has a minimum dependency on Umbraco Forms of `10.0.0`.
+
+To migrate to version 10 you should first update to the latest minor release of version 9. This will ensure you have all the database schema changes in place.
+
+### Breaking changes
+
+Version 10 contains a number of breaking changes but we won't expect many projects to be affected by them as they are in areas that are not typical extension points.  For reference though, the full details are listed here.
+
+#### Database Initialization
+
+When using Umbraco Deploy with Umbraco Cloud, a development database is automatically created when restore a project for the first time into a local environment.  With Umbraco 9 and previous version, SQL CE could be used for this.  This database version is no longer supported in Umbraco 10, so SQLlite is available instead.  SQLlite will be the default format used for the local database.
+
+If you prefer to use a supported alternative, i.e. LocalDb or SQL Server, ensure that a connection string is in place before triggering the restore operation.
+
+For example, to use LocalDb, you would place this in your `appSettings.json` configuration file:
+
+```json
+{
+  "ConnectionStrings": {
+    "umbracoDbDSN": "Data Source=(localdb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Umbraco.mdf;Integrated Security=True",
+    "umbracoDbDSN_ProviderName": "Microsoft.Data.SqlClient"
+  }
+}
+```
+
+#### Configuration
+
+- The superfluous boolean property `IgnoreBrokenDependencies` has been removed, so this option is now controlled only by the configuration key `IgnoreBrokenDependenciesBehavior`, that takes an enumeration value.
+  - The default value for has been changed to `IgnoreBrokenDependenciesBehavior.Restore`, as this will most likely be what developers require (allowing broken dependencies when restoring, but not when pushing to an upstream environment).
```

Comment:
```suggestion
  - The default value has changed to `IgnoreBrokenDependenciesBehavior.Restore`, as this will most likely be what developers require (allowing broken dependencies when restoring, but not when pushing to an upstream environment).
```